platform/network/iot_network_mbedtls.c

```diff
@@ -32,6 +32,15 @@
 #include <stdio.h>
 #include <string.h>
 
+#if defined( __GNUC__ )
```

Comment:
Would probably be better to use OS-specific (instead of compiler defines).

```suggestion
#ifndef( _WIN32 )
```

(Windows being the only OS we support that needs different headers)

Reply:
OK will change that, was not sure which one to use.